src/test/java/com/flightstats/hub/dao/aws/S3WriteQueueTest.java

```diff
@@ -0,0 +1,118 @@
+package com.flightstats.hub.dao.aws;
+
+import com.flightstats.hub.dao.aws.writeQueue.WriteQueueConfig;
+import com.flightstats.hub.metrics.StatsdReporter;
+import com.flightstats.hub.model.ChannelContentKey;
+import com.flightstats.hub.model.Content;
+import com.flightstats.hub.model.ContentKey;
+import com.flightstats.hub.spoke.SpokeWriteContentDao;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+@Slf4j
+public class S3WriteQueueTest {
+
+    private @Mock SpokeWriteContentDao spokeWriteContentDao;
+    private @Mock S3SingleContentDao s3SingleContentDao;
+    private @Mock StatsdReporter statsdReporter;
+    private @Mock ChannelContentKey key;
+    private @Mock ContentKey contentKey;
```

Comment:
Hmm.  @Mock should be before private keyword. Here is the example https://static.javadoc.io/org.mockito/mockito-core/2.27.0/org/mockito/Mock.html
The order for class modifiers as per JLS is here https://docs.oracle.com/javase/specs/jls/se11/html/jls-8.html#jls-8.1.1